server/historian/packages/historian-base/src/services/restGitService.ts

```diff
@@ -238,16 +240,31 @@ export class RestGitService {
     }
 
     public async getSummary(sha: string, useCache: boolean): Promise<IWholeFlatSummary> {
-        return this.resolve(
-            // Currently, only "container" type summaries are retrieved from storage.
-            // In the future, we might want to also retrieve "channels". When that happens,
-            // our APIs will change so we specify what type we want to retrieve during
-            // the request.
-            this.getSummaryCacheKey("container"),
-            async () => this.get<IWholeFlatSummary>(
-                `/repos/${this.getRepoPath()}/git/summaries/${encodeURIComponent(sha)}`),
-            useCache,
-            true);
+        const summaryFetch = async () => this.get<IWholeFlatSummary>(
+            `/repos/${this.getRepoPath()}/git/summaries/${encodeURIComponent(sha)}`);
+
+        if (sha !== latestSummarySha) {
```

Comment:
do we want to add a `toLower` for safety?

Reply:
Good point! I think it's not needed for now - it's a value we control on the driver, and we have been doing similar comparisons (testing if Sha is latest) on GitRest as well with no issues so far.